docs/examples/forecasting/extended_tutorial.md

```diff
@@ -0,0 +1,1746 @@
+
+All the things we did manually here are done by an internal block called `DataLoader`. It gets as an input the raw dataset (in appropriate format) and the transformation object and it outputs the transformed iterable dataset batch by batch. The only thing that we need to worry about is setting the transformation fields correctly! 
+
+
+```python
+
+```
+
+# 3. Training an existing model
+
+As we already mentioned, GluonTS comes with a number of pre-built models that can be used directly with minor hyperparameter configurations. For starters we will use one of these predefined models to go through the whole pipeline of training a model, predicting, and evaluating the results. 
+
+GluonTS gives focus (but is not restricted) to probabilistic forecasting, i.e., forecasting the future distribution of the values and not the future values themselves (point estimates) of a time series. Having estimated the future distribution of each time step in the forecasting horizon, we can draw a sample from the distribution at each time step and thus create a "sample path" that can be seen as a possible realization of the future. In practice we draw multiple samples and create multiple sample paths which can be used for visualization, evaluation of the model, to derive statistics, etc.
+
+In this example we will use a simple pre-built feedforward neural network estimator that takes as input a window of length `context_length` and predicts the distribution of the values of the subsequent future window of length `prediction_length`. 
+
+In general, each estimator (pre-built or custom) is configured by a number of hyperparameters that can be either common (but not binding) among all estimators (e.g., the `prediction_length`) or specific for the particular estimator (e.g., number of layers for a neural network or the stride in a CNN).
+
+Finally, each estimator is configured by a `Trainer`, which defines how the model will be trained i.e., the number of epochs, the learning rate, etc.
+
+
+```python
+from gluonts.model.simple_feedforward import SimpleFeedForwardEstimator
+from gluonts.trainer import Trainer
+```
+
+
+```python
+estimator = SimpleFeedForwardEstimator(
+    num_hidden_dimensions=[10],
+    prediction_length=custom_ds_metadata['prediction_length'],
+    context_length=2*custom_ds_metadata['prediction_length'],
+    freq=custom_ds_metadata['freq'],
+    trainer=Trainer(ctx="cpu", 
+                    epochs=5, 
+                    learning_rate=1e-3, 
+                    hybridize=False, 
+                    num_batches_per_epoch=100,
+                   ),
+)
+```
+
+After specifing our estimator with all the necessary hyperparameters we can train it using our training dataset `dataset.train` just by invoking the `train` method of the estimator. The training returns a predictor that can be used to predict.
+
+
+```python
+predictor = estimator.train(train_ds)
```

Comment:
here, can you add a comment that makes it explicit that a single model (the one defined above) is trained over all time series contained in train_ds, so it's a *global* model.